__docs__/wonder-blocks-link/link-testing-snapshots.stories.tsx

```diff
@@ -61,10 +61,6 @@ type Story = StoryObj<typeof Link>;
  */
 const meta = {
     title: "Packages / Link / Testing / Snapshots / Link",
-    args: {
```

Comment:
I noticed these args were unused when I added a descender to the Primary cell content. Rather than confusing anyone else, I opted to remove unused args.